app/admin/top-issues/components/TopIssueDisplay.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+import { SVGIconChooser } from '@shared/buttons/SVGIconChooser';
+import TextField from '@shared/inputs/TextField';
+import { ButtonGroup } from '@mui/material';
+import Button from '@mui/material/Button';
+import { FaCheck, FaEdit } from 'react-icons/fa';
+import { FaXmark } from 'react-icons/fa6';
+import Image from 'next/image';
+import IconButton from '@mui/material/IconButton';
+import { updateTopIssue } from './TopIssuesList';
+
+export const TopIssueDisplay = ({
+  issue,
+  topIssues = [],
+  setTopIssues = () => {
+  },
+}) => {
+  const [editTopIssueId, setEditTopIssueId] = useState(null);
+  const [editTopIssueName, setEditTopIssueName] = useState(null);
+  const [editTopIssueIcon, setEditTopIssueIcon] = useState(null);
+
+  const handleIssueEdit = issue => () => {
+    setEditTopIssueId(issue.id);
+    setEditTopIssueName(issue.name);
+    setEditTopIssueIcon(issue.icon);
+  };
+
+  const handleClearIssueEdit = () => {
+    setEditTopIssueId(null);
+    setEditTopIssueName(null);
+    setEditTopIssueIcon(null);
+  };
+
+  const handleTopIssueUpdate = (issue) => async () => {
+    const updateIssue = {
+      ...issue,
+      id: editTopIssueId,
+      name: editTopIssueName,
+      icon: editTopIssueIcon,
+    };
+    const { icon } = await updateTopIssue({
+      ...issue,
+      ...updateIssue,
+    });
+    const issueIndx = topIssues.findIndex(({ id }) => id === issue.id);
+    setTopIssues(
+      [
+        ...topIssues.slice(0, issueIndx),
+        {
+          ...updateIssue,
+          ...(icon ? { icon } : {}),
+        },
+        ...topIssues.slice(issueIndx + 1),
+      ],
+    );
+    handleClearIssueEdit();
+  };
+
+  return editTopIssueId === issue.id ?
+    <>
+      {
+        <SVGIconChooser {...{
+          svgData: editTopIssueIcon,
+          setSvgData: data => setEditTopIssueIcon(data),
+        }} />
+      }
+      <TextField {...{
+        className: 'mx-2',
+        fullWidth: true,
+        primary: true,
+        size: 'small',
+        label: 'Top Issue Name',
+        value: editTopIssueName,
+        onChange: (e) => setEditTopIssueName(e.target.value),
+      }} />
+      <ButtonGroup {...{
+        className: 'mr-2',
+        variant: 'outlined',
+      }}>
+        <Button {...{
+          onClick: handleTopIssueUpdate(issue),
+        }}>
+          <FaCheck /> Save
+        </Button>
+        <Button {...{
+          onClick: handleClearIssueEdit,
+        }}>
+          <FaXmark /> Cancel
+        </Button>
+      </ButtonGroup>
+    </> : <>
+      {
+        issue.icon && <Image {...{
+          width: 40,
+          height: 40,
+          src: issue.icon,
+        }} />
+      }
+      <strong>&nbsp; {issue.name}</strong>
+      <IconButton {...{
```

Comment:
Same as above.

Reply:
I couldn't find an edit button in the style guide, like the ones used for the positions that are already in use.  Where in the style guide are these?

<img width="534" alt="image" src="https://github.com/thegoodparty/gp-webapp/assets/496983/e4990a62-a8d1-4c71-8921-e7c355ffae96">


I also didn't think it was very ascthetically pleasing to see even more bright read edit icons in the list, so I just landed on the default MUI `IconButton`here and used the same `FaEdit` icon that the positions edit buttons did. But that's just my subjective view.. 

If you've got an "edit button" style you'd like me to use here, I'd be more than happy to change it if you've got an example?